src/main/webapp/jsp/instructorStudentRecords.jsp

```diff
@@ -1,491 +1,45 @@
-                                        <tr>
-                                            <th colspan="2"> Profile </th>
-                                        </tr>
-                                    </thead>
-                                    <tbody>
-                                        <tr>
-                                            <td class="text-bold">
-                                                Short Name (Gender)</td>
-                                            <td>
-                                                <%= data.studentProfile.shortName.isEmpty() ? "<i class=\"text-muted\">" + Const.STUDENT_PROFILE_FIELD_NOT_FILLED + "</i>" : data.studentProfile.shortName %> (<i><%= data.studentProfile.gender.equals(Const.GenderTypes.OTHER) ? "<span class=\"text-muted\">" + Const.STUDENT_PROFILE_FIELD_NOT_FILLED + "</span>" : data.studentProfile.gender %></i>)
-                                            </td>
-                                        </tr>
-                                        <tr>
-                                            <td class="text-bold">
-                                                Personal Email
-                                            </td>
-                                            <td>
-                                                <%= data.studentProfile.email.isEmpty() ? "<i class=\"text-muted\">" + Const.STUDENT_PROFILE_FIELD_NOT_FILLED + "</i>" : data.studentProfile.email %>
-                                            </td>
-                                        </tr>
-                                        <tr>
-                                            <td class="text-bold">
-                                                Institution
-                                            </td>
-                                            <td>
-                                                <%= data.studentProfile.institute.isEmpty() ? "<i class=\"text-muted\">" + Const.STUDENT_PROFILE_FIELD_NOT_FILLED + "</i>" : data.studentProfile.institute %>
-                                            </td>
-                                        </tr>
-                                        <tr>
-                                            <td class="text-bold">
-                                                Nationality
-                                            </td>
-                                            <td>
-                                                <%= data.studentProfile.nationality.isEmpty() ? "<i class=\"text-muted\">" + Const.STUDENT_PROFILE_FIELD_NOT_FILLED + "</i>" : data.studentProfile.nationality %>
-                                            </td>
-                                        </tr>
-                                    </tbody>
-                                </table>
-                            </div>
+</c:set>
+<c:set var="bodyTitle" value="${data.studentName}'s Records<small class=\"muted\"> - ${data.courseId}</small>" />
```

Comment:
Maybe just use opening and closing `c:set` tags to avoid the use of escaping `\`?